Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "training_modpack"
-version = "3.1.0"
+version = "3.2.0"
```

Comment:
Please keep this at 3.1.0 so we can track-- 3.2.0 betas should be after 3.2.0 releases.

Reply:
Gotcha, this makes sense. Will do.